x-pack/plugins/security_solution/public/timelines/components/timeline/unified_components/data_table/custom_timeline_data_grid_body.tsx

```diff
@@ -117,24 +117,14 @@ const CustomGridRowCellWrapper = styled.div.attrs<{
   className: `rowCellWrapper ${props.className ?? ''}`,
   role: 'row',
 }))`
-  display: flex;
-  align-items: center;
   height: ${(props: { $cssRowHeight: string }) => props.$cssRowHeight};
   .euiDataGridRowCell,
   .euiDataGridRowCell__content {
-    align-items: flex-start;
```

Comment:
This was causing problems for the ellipsis, but also wasn't aligned with the behavior of how the default table functioned